docs/src/main/asciidoc/transaction.adoc

```diff
@@ -307,6 +307,14 @@ The node name identifier needs to be unique per transaction manager deployment.
 And the node identifier needs to be stable over the transaction manager restarts.
 
 The node name identifier may be configured via the property `quarkus.transaction-manager.node-name`.
+[NOTE]
+====
+The node-name property has a length limitation, it cannot be longer than 28 bytes.
+
+The length limitation can possibly be solved by enabling the property 'quarkus.transaction-manager.shortenNodeNameIfNecessary' which, when enabled, will shorten names longer than 28 bytes.
+The shortening process is done by applying the 'SHA-224' algorithm, encoding to Base64 and then truncating the result. Therefore this may result in the node-name ending as an unpredictable duplicate string and this breaks the uniqueness that is mandatory for safe transaction usage.
+For more information about the potential for conflict see https://github.com/quarkusio/quarkus/issues/30491  
```

Comment:
I would simplify the phrasing and would stress that collisions are very unlikely -- see suggestion below, but up to you.

More importantly, Quarkus automatically uses a hyphen-delimited format instead of camel case in `application.properties`, so the actual name of the property is `quarkus.transaction-manager.shorten-node-name-if-necessary`, and you need to fix that (I did below).

```suggestion
The node name cannot be longer than 28 bytes.
To automatically shorten names longer than 28 bytes, set `quarkus.transaction-manager.shorten-node-name-if-necessary` to `true`.

Shortening is implemented by hashing the node name, encoding the hash to Base64 and then truncating the result. As with all hashes, the resulting shortened node name could potentially conflict with another shortened node name, but it is https://github.com/quarkusio/quarkus/issues/30491#issuecomment-1537247764[very unlikely].
```